src/PAModel/Serializers/SourceSerializer.cs

```diff
@@ -368,10 +368,6 @@ private static void GetLogoFile(this CanvasDocument app)
                     app._unknownFiles.Remove(key);
                     app._logoFile = logoFile;
                 }
-                else
-                {
-                    throw new InvalidOperationException($"Missing logo file {key}");
```

Comment:
You may want to consider logging something to our debug logs in this scenario to make sure we're aware of the odd scenario if we get a CRI in the future...

Reply:
No internal debug logging for this repo